config/manager/persistent-volume-claim.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: repo-pvc
+spec:
+  storageClassName: manual
```

Comment:
will there be already a storage class with name as `manual` ? @juliev0

Reply:
Oh, I found an open issue related to it [here](https://github.com/kubernetes/website/issues/30912)

Reply:
https://github.com/kubernetes/website/issues/30912#issuecomment-1828679721